src/extension/mapml/src/test/java/org/geoserver/mapml/MapMLWMSTest.java

```diff
@@ -1590,6 +1590,47 @@ public void testLargeBounds() throws Exception {
                 alternateLinks, ProjType.CBMTILE, new Envelope(-8.1E6, 8.3E6, -3.6E6, 1.23E7), 1e5);
     }
 
+    @Test
+    public void testLicenseIsPopulated() throws Exception {
```

Comment:
Good, but I think this test would have passed even before the fix... (or not?).
What I saw was the MapMLLayerConfig entering the wrong keys in the metadata map, which were then not used by the builder. I believe the actual fix is the change in the constant definitions, although consistent usage of contacts across the code is a very welcomed plus.

Reply:
Good point.  I don't want to create a test that validates a specific key value in case there is a future change but I think this is the next best thing.